sql/core/src/test/resources/sql-tests/results/interval-display-iso_8601.sql.out

```diff
@@ -16,6 +16,6 @@ SELECT
   interval '1 years 2 months -3 days 4 hours 5 minutes 6.789 seconds',
   - interval '1 years 2 months -3 days 4 hours 5 minutes 6.789 seconds'
 -- !query 0 schema
-struct<CAST(NULL AS INTERVAL):interval,0 seconds:interval,1 years:interval,1 months:interval,1 years 2 months:interval,1 days -1 hours:interval,-1 days -1 hours:interval,-1 days 1 hours:interval,-1 days 1 hours:interval,1 years 2 months -3 days 4 hours 5 minutes 6.789 seconds:interval,-1 years -2 months 3 days -4 hours -5 minutes -6.789 seconds:interval>
+struct<CAST(NULL AS INTERVAL):interval,INTERVAL '0 seconds':interval,INTERVAL '1 years':interval,INTERVAL '1 months':interval,INTERVAL '1 years 2 months':interval,INTERVAL '1 days -1 hours':interval,INTERVAL '-1 days -1 hours':interval,INTERVAL '-1 days 1 hours':interval,INTERVAL '-1 days 1 hours':interval,INTERVAL '1 years 2 months -3 days 4 hours 5 minutes 6.789 seconds':interval,INTERVAL '-1 years -2 months 3 days -4 hours -5 minutes -6.789 seconds':interval>
```

Comment:
in this case, we may need fix the `--SET ` issue first

Reply:
you are right. I'll regenerate these after your fix is merged.